### PR TITLE
Convert-Xml: Expose XslCompiledTransform parameter set

### PR DESCRIPTION
**Pull request by vercellone**
Convert-Xml: Expose XslCompiledTransform parameterset to allow for different XsltSettings and/or XmlResolver. The alternative would be to expose these individual settings, but this makes more sense (to me) in terms of long term maintenance; Also supports re-use of the same XslCompiledTransform for better performance;
Fix Set-XmlContent -Formatted

### CODE REVIEW

File: Xml.psm1 [hunk 1]
```diff
@@ -634,44 +635,48 @@ function ConvertFrom-Html {
...
 }
 
-
 function Convert-Xml {
     #.Synopsis
     #   The Convert-XML function lets you use Xslt to transform XML strings and documents.
     #.Description
     #   Documentation TODO
-    [CmdletBinding(DefaultParameterSetName="Xml")]
+    [CmdletBinding(DefaultParameterSetName="Xsl")]
```
Comment (Jaykul) [inline Xml.psm1:643]: Why would we change the default parameter set?

Comment (vercellone) [inline Xml.psm1:643]: I felt Xsl vs. XslCompiledTransform conveyed more than Xml vs. XslCompiledTransform.  "Xml" seemed like a (probable cut and paste) carryover from other functions in the module.  But, the name is functionally insignificant.  If you prefer, leave it "Xml" or call it "JohnBigBooty".  What matters is that I've added an alternate, distinct parametersetname: "XslCompiledTransform".

Comment (Jaykul) [inline Xml.psm1:643]: It's not the name I objected to, it looked like the parameter set that had been default was now secondary. After reading it a little more closely, that's not the case, I'll merge this.

Comment (vercellone) [inline Xml.psm1:643]: Just a misunderstanding.  As you discovered this commit included no breaking changes (I learned my lesson!).  Thanks for circling back.

**Pull request merged**